libvast/test/address.cpp

```diff
@@ -146,3 +148,230 @@ TEST(parseable) {
   CHECK(a.is_v6());
   CHECK(to_string(a) == str);
 }
+
+TEST(IPv4 pseudonymization - seed 1) {
+  std::array<vast::address::byte_type, 32> seed
+    = {21,  34,  23,  141, 51,  164, 207, 128, 19, 10, 91, 22, 73, 144, 125, 16,
+       216, 152, 143, 131, 121, 121, 101, 39,  98, 87, 76, 45, 42, 132, 34,  2};
+  // test data from
+  // https://github.com/noinkling/node-cryptopan/blob/main/src/test/test_data.ts
```

Comment:
No request for changes, but thank you so much for writing extensive tests. This is really nice to see!

Reply:
:+1: